docs/framework/additional-apis/s_servicepointtable.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "s_ServicePointTable Field | Microsoft Docs"
+ms.date: "04/24/2017"
+ms.prod: ".net-framework"
+ms.technology: ""
+ms.topic: "reference"
+topic_type: "apiref"
+api_name:
+  - "System.Net.ServicePointManager.s_ServicePointTable"
+api_location:
+  - "System.dll"
+api_type: "Assembly"
+ms.assetid: 24459679-291c-401a-9def-e42b29466fcf
+author: "guardrex"
+ms.author: "mairaw"
+manager: "wpickett"
+---
+# s_ServicePointTable Field
+
+`ServicePointManager.s_ServicePointTable` is a <xref:System.​Collections.​Hashtable> that contains the list of active HTTP connections (<xref:System.​Net.​Service​Point>s) in the <xref:System.​App​Domain>. 
```

Comment:
Probably a separate problem; but when I paste `System.​Collections.​Hashtable` into the API Browser search box, it's coming back "No results." When I type in the ref starting with "System" it works.

Let's see what happens now that I'm fixing the metadata and re-building soon.